src/core/Header/HeaderLogo.tsx

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import '@itwin/itwinui-css/css/header.css';
+
+import cx from 'classnames';
+import React from 'react';
+
+import { useTheme } from '../utils/hooks/useTheme';
+import { CommonProps } from '../utils/props';
+
+export type HeaderLogoProps = React.PropsWithChildren<
```

Comment:
Would be better to define `children` explicitly rather than use `PropsWithChildren` so that we can add a JSDoc comment to it.

Reply:
Added for `HeaderLogo` and `Header` with positionning information, couldnt find better to put in there.